pycloudlib/oci/instance.py

```diff
@@ -76,17 +76,23 @@ def ip(self):
             vnic_attachment = self.compute_client.list_vnic_attachments(
                 compartment_id=self.compartment_id,
                 instance_id=self.instance_data.id,
-            ).data[0]
-            vnic_info = self.network_client.get_vnic(vnic_attachment.vnic_id).data
+            ).data
+            vnics = [
+                self.network_client.get_vnic(vnic_attachment.vnic_id).data
+                for vnic_attachment in vnic_attachment
```

Comment:
Just for readability, lets name the loop-local variable name vnic instead of vnic_attachment. I have a hard time seeing the for loop use the same iterator variable name as the list it is walking.

```suggestion
                self.network_client.get_vnic(vnic.vnic_id).data
                for vnic in vnic_attachment
```

Reply:
The naming is more so to make it clear that the thing we are iterating with is not actually a vnic but instead a vnic attachment from which we can call a function and get the actual vnic. but that being said i don't actually care that much and would be happy to rename to `vnic` if you think that is better for readability!